src/devices/screen.c

```diff
@@ -293,10 +456,13 @@ static inline bool _touch_status_equivalent(V5_TouchStatus x, V5_TouchStatus y)
 	return (x.lastEvent == y.lastEvent) && (x.lastXpos == y.lastXpos) && (x.lastYpos == y.lastYpos);
 }
 
-void _touch_handle_task(void* ignore) {
+uint32_t _touch_handle_task(void* ignore) {
```

Comment:
Given that this is a daemon for handling touch callbacks, I don't think we really need this to be changed to have a return value.